statsmodels/base/model.py

```diff
@@ -1055,7 +1055,14 @@ def llf(self):
 
     @cache_readonly
     def bse(self):
-        return np.sqrt(np.diag(self.cov_params()))
+        # Issue 3299
+        if ((not hasattr(self, 'cov_params_default')) and
+                (self.normalized_cov_params is None)):
+            bse_ = np.empty(len(self.params))
+            bse_[:] = np.nan
```

Comment:
FYI for future: np.full to create an array with a selected fill value
But I don't remember when it was added, and the numpy docstring doesn't say.